src/libs/actions/Session/index.js

```diff
@@ -289,6 +289,46 @@ function resetPassword() {
     API.write('RequestPasswordReset', {
         email: credentials.login,
     },
+    {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.ACCOUNT,
+                value: {
+                    isLoading: true,
```

Comment:
NAB maybe this is just me, but it looks a bit odd to be redirected to the Forgot page and see the loading indicator when the message above says we sent a link already. Should we just remove the loading indicator from the first load? 🤔

Reply:
Yep agreed.